### DISCUSSION TITLE
docker overlay directory fills up fast due to suggester related files which causes all the indexer process to fail

### DISCUSSION THREAD

**Discussion by jkjha** [oracle/opengrok#4252] [Q&A]
We are running docker as webapp and the source and data directory mounted over volume mounts.

However, we've been experiencing issues as we add more projects to our instance. The default storage for our Docker pods is 5GB, which gets filled up within 1-2 days, causing the pod to crash and preventing any new processes from running.

On further debugging found that `/usr/local/tomcat/temp/` is growing rapidly with some suggester related data. 

I think its getting cleaned up but growing faster than that. I see similar issue reported earlier but those are closed...

<img width="654" alt="image" src="https://user-images.githubusercontent.com/4188259/228572941-76cabc17-de80-49ab-9820-68eddd013477.png">

**Comment by vladak** [comment]
The leak is tracked by #2857, however I think this is a bug in Lucene, not in OpenGrok.

Anyhow, the Tomcat temporary directory in the Docker container should probably reside under `/opengrok/data`.

**Reply by vladak** [reply]
Will change this with PR #4260.

**Reply by jkjha** [reply]
Thanks again. I see you already have fix ready here https://github.com/oracle/opengrok/pull/4260 ... any idea when it can be merged to master?

**Reply by vladak** [reply]
I asked my colleague for review, once that's done it will be merged in.
